tests/utils/command.go

```diff
@@ -76,16 +102,47 @@ func RegisterNodeRun() {
 
 		// Assumes that startCmd will launch a node with HTTP Port at [utils.DefaultLocalNodeURI]
 		healthClient := health.NewClient(DefaultLocalNodeURI)
-		healthy, err := health.AwaitReady(ctx, healthClient, 5*time.Second, nil)
+		healthy, err := health.AwaitReady(ctx, healthClient, healthCheckTimeout, nil)
 		gomega.Expect(err).Should(gomega.BeNil())
 		gomega.Expect(healthy).Should(gomega.BeTrue())
 		log.Info("AvalancheGo node is healthy")
+
+		subnets := make(map[string]string)
+		files, err := filepath.Glob("./tests/precompile/genesis/*.json")
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		for _, file := range files {
+			basename := filepath.Base(file)
+			index := basename[:len(basename)-5]
+			subnets[index] = CreateNewSubnet(ctx, file)
```

Comment:
can we create those subnets in parallel? i.e in their old places?

Reply:
No, because they share the same wallet and if we do that we will have race conditions.